src/lv_draw/lv_draw_blend.c

```diff
@@ -352,7 +352,8 @@ LV_ATTRIBUTE_FAST_MEM static void fill_normal(const lv_area_t * disp_area, lv_co
 
 #if LV_USE_GPU_STM32_DMA2D
             if(lv_area_get_size(draw_area) >= 240) {
-                static lv_color_t blend_buf[LV_HOR_RES_MAX] = {0};
+                /* blend_buf MUST be in DMA accessible RAM (depending on linker this isn't always the case - CCMRAM) */
+                LV_DMA_ATTR static lv_color_t blend_buf[LV_HOR_RES_MAX] = {0};
```

Comment:
Now that I look at it, do we need separate `blend_buf` variables for these two conditions? What do you think about moving this to the top of the file?

Reply:
I can move it to top of file no problem.

Reply:
We need to be sure that it's not declared if not used. On an 800x480 32 bit screen, it could mean extra 3200 byte RAM.

Reply:
It can be wrapped in an ifdef